testsuite/features/secondary/srv_manage_activationkey.feature

```diff
@@ -0,0 +1,99 @@
+# Copyright (c) 2010-2021 SUSE LLC
+# Licensed under the terms of the MIT license.
+￼
+Feature: Manipulate activation keys
+  In order to register systems to the spacewalk server
+  As the testing user
+  I want to create and edit activation keys
+
+  Scenario: Create an activation key for i586
+    Given I am on the Systems page
+    When I follow the left menu "Systems > Activation Keys"
+    And I follow "Create Key"
+    And I enter "SUSE Test Key i586" as "description"
+    And I enter "SUSE-TEST-i586" as "key"
+    And I check "virtualization_host"
+    And I click on "Create Activation Key"
+    Then I should see a "Activation key SUSE Test Key i586 has been created." text
+    And I should see a "Details" link
+    And I should see a "Packages" link
+    And I should see a "Configuration" link in the content area
+    And I should see a "Groups" link
+    And I should see a "Activated Systems" link
```

Comment:
We already test those links in core.

Reply:
you are right - it was an effect of brute cut and paste :-) . Removed